Makefile

```diff
@@ -0,0 +1,5 @@
+.PHONY: start
+
+start:
+	mix deps.get
+	iex -S mix phx.server
```

Comment:
It's not needed now, but we're not running the docker-compose files for each app here.